javascript-create-module/template/package.json

```
@@ -17,42 +15,44 @@
     "format": "prettier --write --list-different .",
     "lint": "eslint .",
     "package": "yarn pack --out dist/package.tgz",
-    "watch": "vite build --watch",
+    "dev": "vite build --watch",
+    "watch": "run dev",
     "watch:callback": "run package && run deploy"
   },
   "dependencies": {
+    "canvas-confetti": "^1.9.3",
+    "clsx": "^2.1.1",
     "modern-normalize": "^3.0.1"
   },
   "devDependencies": {
     "@eslint-react/eslint-plugin": "^1.26.2",
     "@eslint/compat": "^1.2.6",
     "@eslint/js": "^9.20.0",
-    "@jahia/javascript-modules-library": "0.x || ^1.0.0",
-    "@jahia/vite-plugin": "^0.5.1",
+    "@jahia/javascript-modules-library": "latest",
+    "@jahia/vite-plugin": "latest",
+    "@types/canvas-confetti": "^1.9.0",
     "@types/react": "^19.0.0",
     "eslint": "^9.20.1",
     "eslint-plugin-cypress": "^4.1.0",
     "globals": "^15.15.0",
     "i18next": "^23.10.1",
     "prettier": "^3.5.1",
-    "prettier-plugin-jsdoc": "^1.3.2",
-    "prettier-plugin-packagejson": "^2.5.8",
     "react-i18next": "^15.4.0",
     "typescript": "^5.7.3",
     "typescript-eslint": "^8.24.0",
+    "typescript-plugin-css-modules": "^5.1.0",
     "vite": "^6.1.0"
   },
   "packageManager": "yarn@4.7.0",
   "engines": {
-    "node": ">=18.0.0",
+    "node": ">=22.0.0",
     "yarn": ">=4.0.0"
   },
   "jahia": {
     "snapshot": true,
     "module-dependencies": "default",
-    "module-type": "$$MODULE_TYPE$$",
-    "module-type-comment": "Use templatesSet in the module type to declare a template set",
+    "module-type": "templatesSet",
     "server": "dist/server/index.js",
-    "static-resources": "/icons,/images,/javascript,/locales"
+    "static-resources": "/static,/dist,/locales,/images,/icons"
```

`/static` is always added to the list of `static-resources` ([here](https://github.com/Jahia/javascript-modules/blob/main/javascript-modules-engine-java/src/main/java/org/jahia/modules/javascript/modules/engine/jshandler/JavascriptProtocolConnection.java#L215)):
```suggestion
    "static-resources": "/dist,/locales,/images,/icons"
```

---

Good to know! Why???